product/index.md

```diff
@@ -2,6 +2,17 @@
 
 This covers how we run our business and deliver service to our customers.
 
+## Product Vision
+Our target customers are users familiar with Node-RED looking to run multiple instances or share access across multiple users.
+
+We make it easy to create instances of Node-RED and to share configuration and work between them. We provide collaboration functionality allowing multiple people in a team to build and maintain a flow. 
+We give user a management platform that delivers the features around running node-red such that it requires minimal effort on the part of the customer to configure, update, and maintain their environments.
+We enable customers to run the flows wherever they require based on what systems and devices they are connecting to.
```

Comment:
More generally; where ever Node-RED runs, FlowForge will support you? Going back to @knolleary's remark yesterday that FlowForge is the answer to running Node-RED in production.

Reply:
This point was a specific reference to our capability for both server and device side instances, I like the point that its about what you are connecting to that determines where you need to run Node-RED